File: accepted-solutions/fizzbuzz/fizzbuzz.py
def main():
    nums = [int(i) for i in input().split()]
    
    for i in range(nums[2]):
        if (i + 1) % nums[0] == 0:
            if (i + 1) % nums[1] == 0:
                print("Fizzbuzz")
            else:
                print("Fizz")
        
        elif (i + 1) % nums[1] == 0:
            print("Buzz")
        
        else:
            print(i + 1)

if __name__ == "__main__":
    main()

File: accepted-solutions/fizzbuzz/fizzbuzz.java
import java.util.Scanner;

public class fizzbuzz {
    public static void main(String[] args) {
        Scanner cin = new Scanner(System.in);
        int x = cin.nextInt();
        int y = cin.nextInt();
        int n = cin.nextInt();

        for (int i = 1; i <= n; ++i) {
            if (i % x == 0) {
                if (i % y == 0) {
                    System.out.println("Fizzbuzz");
                }
                else {
                    System.out.println("Fizz");
                }
            }

            else if (i % y == 0) {
                System.out.println("Buzz");
            }

            else {
                System.out.println(i);
            }
        }
    }
}
